ios/bridge/NRMModularAgent.m

```diff
@@ -38,7 +38,6 @@ - (dispatch_queue_t)methodQueue{
                   config:(NSDictionary* _Nullable)agentConfig
                   startWithResolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject){
-    NSLog(@"NRMA calling start agent for RN bridge is deprecated. The agent automatically starts on creation.");
```

Comment:
removed this line since we are calling start agent on iOS devices

Reply:
ok